Guides/Getting Started.md

```diff
@@ -1,3 +1,128 @@
 # Getting Started
 
-> TODO:
+ReactiveLists provides a React-like API for `UITableView` and `UICollectionView`.  This goal is to provide a more
+declarative interface on top of your normal list and collection code.  To get started with ReactiveLists, we encourage you to play around with the examples included in the repository, but here we will also discuss the main components of the library.
+
+To get set up, you first need to add a `Driver` (either a `TableViewDriver` or `CollectionViewDriver`) to your view controller.
```

Comment:
I think all of the explanation should come first. *then* example code.

Reply:
Do you mean to not mix the text explaining the code with the example code?  Or do you just mean pulling up the `Glossary of Types` (soon to be renamed :) ) above this section with code?